pkg/kubectl/cmd/cp_test.go

```diff
@@ -100,6 +100,7 @@ func TestGetPrefix(t *testing.T) {
 func TestTarUntar(t *testing.T) {
 	dir, err := ioutil.TempDir(os.TempDir(), "input")
 	dir2, err2 := ioutil.TempDir(os.TempDir(), "output")
+	dir = dir + "/"
```

Comment:
For other people looking at this place add a comment explaining why this is here.